DigitalLearningSolutions.Web.Tests/TestHelpers/ListTestHelper.cs

```diff
@@ -0,0 +1,21 @@
+﻿namespace DigitalLearningSolutions.Web.Tests.Helpers
+{
+    using System.Collections.Generic;
+    using System.Linq;
+
+    public static class ListTestHelper
+    {
+        public static bool ListOfStringsMatch(
+            List<string> list1,
+            List<string> list2
+        )
+        {
+            if (list1.Count != list2.Count)
+            {
+                return false;
+            }
+
+            return list1.All(list2.Contains);

```

Comment:
This doesn't quite do what you want it to do - it'll return true if list 1 is {"1", "1"} and list 2 is {"1", "2"}.
I think you want the first option from https://www.techiedelight.com/check-whether-two-lists-have-the-same-items-in-csharp/